src/cli.js

```diff
@@ -11,10 +11,52 @@ import fs from 'fs';
 import ciChoices from '../ci-choices.json';
 import containerizationChoices from '../containerization-choices.json';
 
+function initDocker(config) {
+  console.log('Creating Dockerfile and .dockerignore...');
+
+  const dockerFileContents = `# Use the official Node runtime as a parent image
+# More info at https://hub.docker.com/_/node/
+FROM node:alpine
+    
+# Set the working directory
+WORKDIR /usr/src/app
+
+# Install app dependencies
+COPY package.json package-lock.json ./
+RUN npm install
+
+# Bundle app source
+COPY . .
+
+# Make port 8080 available to the world outside this container
+EXPOSE 8080
+
+# Run "npm start" when the container launches
+CMD ["npm", "start"]
+`;
```

Comment:
can we make this a template file?

Reply:
What's the node way of doing that?

Reply:
by template file, I mean just its own file